code/unclassified/src/leap_year/leap_year.js

```diff
@@ -0,0 +1,11 @@
+
+const isLeap = year => ((year % 4 == 0) && (year % 100 != 0)) || (year % 400 == 0)
+
+let startYear = 1990
+let endYear = 2020
+
+for(let i = startYear; i<= endYear; i++) {
```

Comment:
One more change. Add a space before the opening parentheses here and before the first one on the line below. Additionally, use pre-increment over post increment.